cln-rpc/examples/getinfo.rs

```diff
@@ -1,13 +1,11 @@
 use anyhow::Context;
 use cln_rpc::{model::GetinfoRequest, ClnRpc, Request};
-use log::info;
 use std::env::args;
 use std::path::Path;
 use tokio;
 
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
-    env_logger::init();
```

Comment:
Pretty sure this needs to stay where it is, otherwise we don't get any logs.

Reply:
ops! do you mean the log inside the library?